service/resource/configmap/service_test.go

```diff
@@ -4,12 +4,23 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/giantswarm/ingress-operator/tests"
 	"github.com/giantswarm/ingresstpr"
 	"github.com/giantswarm/ingresstpr/spec"
 	"github.com/giantswarm/ingresstpr/spec/hostcluster"
 	apiv1 "k8s.io/client-go/pkg/api/v1"
 )
 
+func testConfig(t *testing.T) Config {
```

Comment:
This is needed because if not the `DefaultConfig` method tries to instantiate a k8s client with `config.Incluster` set to true. Also, there are separate `testConfig` for service and configmap because the Config types in each of those packages are also different (although they contain the same fields).

Reply:
I don't think K8s client is actually used in those tests. Did you try running that w/o creating `testConfig`?

Also `microloggertest.New()` come in handy here: https://github.com/giantswarm/micrologger/blob/master/microloggertest/logger.go#L10

Reply:
+1 for using `microloggertest.New()`